my-app/src/services/history.js

```diff
@@ -0,0 +1,56 @@
+export const getData = () => Promise.resolve([
+    {
+        id: 1,
+        admission_date: '12.12.2019',
```

Comment:
tutaj podobnie. Trzymajmy się konwencji `airBnb`  (popularne): https://github.com/airbnb/javascript#naming--camelCase
(to powinien być `camelCase`, tj `admissionDate`)

Reply:
Oczekują na zmianę :)

* `admission_date`
* `discharge_date`
* `patients_condition`